policy/release/test/test.rego

```diff
@@ -257,7 +257,8 @@ deny contains result if {
 	img := image.parse(input.image.ref)
 	img_digest := img.digest
 
-	some task in lib.images_processed_results_from_tests
```

Comment:
Do we use this library anywhere else or can we remove it wholesale?

Reply:
I moved it inside the other rule, and also to the policy file.